packages/visx-xychart/src/utils/getBarStackRegistryData.ts

```diff
@@ -1,15 +1,17 @@
 import { AxisScale } from '@visx/axis';
-import { getSecondItem } from '@visx/shape/lib/util/accessors';
+import { getFirstItem, getSecondItem } from '@visx/shape/lib/util/accessors';
 import { extent } from 'd3-array';
 import { BarStackData, BarStackDatum, DataRegistryEntry } from '../types';
 
 const getStack = <XScale extends AxisScale, YScale extends AxisScale>(
   bar: BarStackDatum<XScale, YScale>,
 ) => bar?.data?.stack;
 
+// returns average of top + bottom of bar (the middle) as this enables more accurately
+// finding the nearest datum to a FocusEvent (which is based on the middle of the rect bounding box)
 const getNumericValue = <XScale extends AxisScale, YScale extends AxisScale>(
   bar: BarStackDatum<XScale, YScale>,
-) => getSecondItem(bar); // corresponds to y1, the upper value (topline).
+) => (getFirstItem(bar) + getSecondItem(bar)) / 2;
```

Comment:
when only accounting for the top line of a bar, logic for finding the nearest series datum to the middle of the bar bounding box was incorrect